### PR TITLE
[ipcamera] Add support to change MJPEG source to a different RTSP sub stream to lower CPU usage

### PR DESCRIPTION
**Pull request by Skinah**
Adds support to use a lower resolution RTSP feed as the mjpeg source which will lower CPU load on the openHAB server. Previously if you used a lower res feed, then any recordings made would use the lower res stream as well as the mjpeg feed.

### CODE REVIEW

File: bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java [hunk 1]
```diff
@@ -811,7 +811,7 @@ public void setupFfmpegFormat(FFmpegFormat format) {
             logger.warn("The camera tried to use a FFmpeg feature when the location for FFmpeg is not known.");
             return;
         }
-        if (rtspUri.toLowerCase().contains("rtsp")) {
+        if (mjpegUri.toLowerCase().startsWith("rtsp") || rtspUri.toLowerCase().startsWith("rtsp")) {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java:814]: Should you rather consider "rtsp://"? Later in that method, you use "rtsp://".

File: bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java [hunk 2]
```diff
@@ -1773,7 +1787,7 @@ private boolean supportsOnvifEvents() {
     private void keepMjpegRunning() {
         CameraServlet localServlet = servlet;
         if (localServlet != null && !localServlet.openStreams.isEmpty()) {
-            if (!mjpegUri.isEmpty() && !"ffmpeg".equals(mjpegUri)) {
+            if (!mjpegUri.isEmpty() && !"ffmpeg".equals(mjpegUri) && !mjpegUri.toLowerCase().startsWith("rtsp://")) {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java:1790]: I believe you can use `!usingRtspForMjpeg()`

Comment (Skinah) [reply bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java]: Thanks that is easier to read.

File: bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java (hunk 1 above)
Comment (Skinah) [reply bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java]: Have added the suggestion.

**Review by lsiepel — APPROVED** [review]
LGTM. @lolodomo i'll let you merge this as you started the review.

**Pull request merged**